src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SentReferralWithActionPlanIdDTO.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class SentReferralWithActionPlanIdDTO(
```

Comment:
why do we need this extra class? can't we just have the optional actionPlanId on the `SentReferralDTO`?

Reply:
Will revert back to SentReferralDTO and make change in that